src/rgw/rgw_iam_policy.h

```diff
@@ -371,6 +370,14 @@ struct Condition {
     }
   };
 
+  struct string_like : public std::binary_function<const std::string,
```

Comment:
Just a question: I read here: http://en.cppreference.com/w/cpp/utility/functional/binary_function, that binary_function is deprecated in C++11 and removed in C++17. Do we still want to use it?